frontend/src/components/maps/MineMap.js

```diff
@@ -14,16 +15,28 @@ import * as String from "@/constants/strings";
  */
 const propTypes = {
   mine: PropTypes.object,
+  map: PropTypes.object,
+  view: PropTypes.object,
   lat: PropTypes.number,
   long: PropTypes.number,
 };
 
 const defaultProps = {
   mine: null,
+  map: {},
+  view: {},
+  lat: null,
+  long: null,
 };
 
 class MineMap extends Component {
-  state = { map: null, view: null, center: null, zoom: null };
+  state = {
+    map: this.props.map,
```

Comment:
Looking at `handleLoadMap`, it looks like `map` and `view` are only ever used for state? Are they ever actually passed in as props to this component?

If they aren't then we don't need them in the propTypes.

If they are, you'll have to identify the properties/methods that exist on map and view respectively which this component leverages and update the propTypes to include those properties/methods using `PropTypes.shape()`. `PropTypes.object` is forbidden by our linting rules.

Reply:
So if I leave the center as an empty array, a LocationPin appears on the top left of the map.
My Guess is, an empty list is somehow processed as [0,0] by the mapping library. But if you keep it null, it gets ignored